style/README.md

```diff
@@ -12,22 +12,15 @@ style guide violations.
 In addition to the general guidelines below, we also have the following more
 detailed, language/framework-specific style guides:
 
-* [Android](android)
-* [Backbone](backbone)
 * [CoffeeScript](coffeescript)
-* [Ember.js](ember)
 * [ERb](erb)
```

Comment:
Nutzen wir noch irgendwo ERB oder könnte das auch raus? Sollten wir evtl. in Zukunft durch einen Guide für HAML ersetzen

Reply:
Wird für den Plaintext Mime Part der Mails genutzt und evtl noch ein paar sehr alte Views?

Reply:
Achso, ok